elastic/logstash/dictionaries/action.yml

```diff
@@ -0,0 +1,4 @@
+GET .*/api/mines/[0-9A-Fa-f]{8}(?:-[0-9A-Fa-f]{4}){3}-[0-9A-Fa-f]{12}: Mine View
```

Comment:
We're going to want to add a lot more items here. We should add a reminder to ourselves to talk about this in the readme and in general what events we want to capture.

Reply:
Defs!